master/internal/resourcemanagers/resource_pool.go

```diff
@@ -151,6 +151,14 @@ func (rp *ResourcePool) getOrCreateGroup(
 		return g
 	}
 	g := &group{handler: handler, weight: 1}
+
+	if rp.config.Scheduler.Priority != nil {
+		if rp.config.Scheduler.Priority.DefaultPriority == nil {
+			panic("default priority is not configured")
```

Comment:
q: I thought we settled on a constant value for default priority. why we need to handle the nil case?

Reply:
It isn't configured when using fair_share, and this is just catching if we break that configuration logic somehow down the road. Can also just get rid of this.

Reply:
it's fine to keep it